Signal/src/AppDelegate.m

```diff
@@ -56,7 +56,6 @@
 @interface AppDelegate ()
 
 @property (nonatomic) UIWindow *screenProtectionWindow;
-@property (nonatomic) OWSContactsSyncing *contactsSyncing;
```

Comment:
Convert OWSContactsSyncing to singleton to ensure it's only created once.

Reply:
👍